Argo/Types/JSON.swift

```diff
@@ -9,6 +10,15 @@ public enum JSON {
 }
 
 public extension JSON {
+  /**
```

Comment:
Yep, `/*` is a multi-line comment, `/**` is a multi-line doc comment. This (and the `///` for single line comments) prompt Xcode to parse and display these blocks inside the IDE using a special flavor of markdown.